src/library/basetracktablemodel.h

```diff
@@ -300,7 +301,8 @@ class BaseTrackTableModel : public QAbstractTableModel, public TrackModel {
 
     static int s_bpmColumnPrecision;
     static bool s_keyColorsEnabled;
-    static ColorPalette s_keyColorPalette;
+    // This need to be left uninitialized to avoid static initialization order issues
+    static std::optional<ColorPalette> s_keyColorPalette;
```

Comment:
This comment is misleading IMO. its not uninitialized, it just gets implicitly initialized to its `std::nullopt` state...